<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="load_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>511</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Carga el archivo que deseas clasificar</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="load_file_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar archivo</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_name_label">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>170</y>
      <width>511</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre del archivo cargado</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
